### PR TITLE
[ADD] helpdesk_type_sla: implement rules based on type

### PR DESCRIPTION
**Pull request by BojanOD**
Implement rules based on type that depends on:

- https://github.com/OCA/helpdesk/pull/362

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

File: helpdesk_type_sla/views/helpdesk_sla_views.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<odoo>
+
+    <record model="ir.ui.view" id="helpdesk_ticket_sla_form">
+        <field name="model">helpdesk.sla</field>
+        <field name="inherit_id" ref="helpdesk_mgmt_sla.helpdesk_ticket_sla_form" />
+        <field name="arch" type="xml">
+            <field name="tag_ids" position="after">
+                <field name="type_ids" />
```
Comment (ivantodorovich) [inline helpdesk_type_sla/views/helpdesk_sla_views.xml]:
```suggestion
                <field name="type_ids" widget="many2many_tags" />
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by elvise** [comment]
> Implement rules based on type that depends on:
> 
> * [[IMP] helpdesk_mgmt_sla: implement more advanced rules #362](https://github.com/OCA/helpdesk/pull/362)

@BojanOD with the widget domain in this one #362 i  guess we can close this, right ?

**Comment by ivantodorovich** [comment]
> with the widget domain in this one https://github.com/OCA/helpdesk/pull/362 i guess we can close this, right ?

Not sure. Some unexperienced users don't know how to use the domain widget, they might find it easier to use the field

**Comment by BojanOD** [comment]
Hi @elvise @ivantodorovich 

Yes I guess we can close this PR now. Can you have a look. Thanks

File: helpdesk_type_sla/views/helpdesk_sla_views.xml (hunk 1 above)
Comment (ivantodorovich) [reply helpdesk_type_sla/views/helpdesk_sla_views.xml]: This comment is still pending

File: helpdesk_type_sla/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#. Allow SLA for a Helpdesk's Team
+#. Set a resource calendar
+
+Allow SLA
+~~~~~~~~~~~~~~~
+
+#. Go to Helpdesk > Configuration > SLA.
+#. Edit or create a new SLA.
+#. Check Allow SLA option to allow SLA for that team, category or tag.
+#. Select a days or hours for that SLA.
```
Comment (ivantodorovich) [inline helpdesk_type_sla/readme/CONFIGURE.rst]: You can drop this file, it's already described in `helpdesk_sla`

File: helpdesk_type_sla/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module adds SLA functionality in Helpdesk module.
```
Comment (ivantodorovich) [inline helpdesk_type_sla/readme/DESCRIPTION.rst]: Rewrite this, it's not accurate

**Review by ivantodorovich — APPROVED** [review]

**Comment by BojanOD** [comment]
Hi @elvise , can you review my code. So we can merge this PR. Thank you

**Comment by BojanOD** [comment]
Hi @elvise @pedrobaeza  can we close this PR

**Comment by pedrobaeza** [comment]
You can close it as well clicking on the button at the footer, but why do you want to do that?

**Comment by elvise** [comment]
@BojanOD Sure. 
@pedrobaeza because there is sla with domain widget

**Comment by pedrobaeza** [comment]
OK, closing then.

**Comment by BojanOD** [comment]
> You can close it as well clicking on the button at the footer, but why do you want to do that?

Sorry, I meant can we merge this PR @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
Code review

/ocabot merge nobump

**Pull request merged**